src/test/scala/no/gdl/bookapi/service/search/SearchServiceIntegrationTest.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2018 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.service.search
+
+import com.sksamuel.elastic4s.http.HttpClient
+import io.digitallibrary.language.model.LanguageTag
+import no.gdl.bookapi.integration.E4sClient
+import no.gdl.bookapi.model.domain.{Paging, Sort}
+import no.gdl.bookapi.{TestData, TestEnvironment, UnitSuite}
+import no.gdl.tag.IntegrationTest
+import org.mockito.Mockito.when
+
+@IntegrationTest
+class SearchServiceIntegrationTest extends UnitSuite with TestEnvironment {
+
+  override val searchService = new SearchService
+  override val indexService = new IndexService
+  override val esClient: E4sClient = E4sClient(HttpClient("http://localhost:9200")) // Requires running elasticsearch
+
+  override def beforeAll(): Unit = {
+    when(translationRepository.languagesFor(1)).thenReturn(Seq(LanguageTag(TestData.DefaultLanguage)))
+    when(bookRepository.withId(1)).thenReturn(Some(TestData.Domain.DefaultBook))
+    when(converterService.toApiBook(Some(TestData.Domain.DefaultTranslation), Seq(LanguageTag(TestData.DefaultLanguage)), Some(TestData.Domain.DefaultBook))).thenReturn(Some(TestData.Api.DefaultBook))
+    val translation = TestData.Domain.DefaultTranslation
+    indexService.indexDocument(translation)
+  }
+
+  test("that search for unknown language gives empty search result") {
+    val searchResult = searchService.searchWithQuery(LanguageTag("aaa"), None, Paging(1,10), Sort.ByIdAsc)
+    searchResult.totalCount should be(0)
+  }
+
+  test("that search for books in norwegian returns just that") {
+    val searchResult = searchService.searchWithQuery(LanguageTag("nob"), None, Paging(1,10), Sort.ByIdAsc)
+    searchResult.totalCount should be(1)
+  }
+
```

Comment:
Denne testen er kanskje litt tynn? Hadde vært fint med noen tester som verifiserer noen enkle søk at vi får forventede svar på hvertfall ett språk som ElasticSearch har støtte for, samt et språk der ICU benyttes.